etl/steps/data/garden/energy/2022-12-13/electricity_mix.py

```diff
@@ -0,0 +1,324 @@
+"""Garden step that combines BP's statistical review with Ember's combined electricity data (combination of the European
+Electricity Review and the Yearly Electricity Data) to create the Electricity Mix (BP & Ember) dataset.
+
+"""
+
+from typing import Dict, List
+
+import pandas as pd
+from owid import catalog
+from owid.datautils.dataframes import combine_two_overlapping_dataframes
+from shared import CURRENT_DIR, add_population
+
+from etl.paths import DATA_DIR
+
+# Details for dataset to export.
+DATASET_SHORT_NAME = "electricity_mix"
+METADATA_PATH = CURRENT_DIR / f"{DATASET_SHORT_NAME}.meta.yml"
```

Comment:
You could use `etl.helpers.Names` here.

```python
from etl.helpers import Names

# naming conventions
N = Names(__file__)

# Details for dataset to export.
DATASET_SHORT_NAME = N.short_name
METADATA_PATH = N.metadata_path
```

Reply:
True, but I'll avoid these changes for now.